tests/suite/test_v_s_route_redirects.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+import requests
+
+
+from settings import TEST_DATA
+from suite.custom_assertions import assert_event_and_get_count, wait_and_assert_status_code, \
+    assert_event_count_increased, assert_event_starts_with_text_and_contains_errors
+from suite.custom_resources_utils import get_vs_nginx_template_conf, patch_v_s_route_from_yaml
+from suite.resources_utils import get_first_pod_name, get_events, wait_before_test
+
+
+@pytest.mark.vsr
+@pytest.mark.parametrize('crd_ingress_controller, v_s_route_setup',
+                         [({"type": "complete", "extra_args": [f"-enable-custom-resources"]},
+                           {"example": "virtual-server-route-redirects"})],
+                         indirect=True)
+class TestVSRRedirects:
+    def test_config(self, kube_apis, ingress_controller_prerequisites, crd_ingress_controller, v_s_route_setup):
+        wait_before_test(1)
+        ic_pod_name = get_first_pod_name(kube_apis.v1, ingress_controller_prerequisites.namespace)
+        config = get_vs_nginx_template_conf(kube_apis.v1,
+                                            v_s_route_setup.namespace,
+                                            v_s_route_setup.vs_name,
+                                            ic_pod_name,
+                                            ingress_controller_prerequisites.namespace)
+        assert 'return 307 ' in config and 'return 301 ' in config
+
+    def test_custom_canned_response(self, kube_apis, crd_ingress_controller, v_s_route_setup):
```

Comment:
hm, the name is for canned responses but we're testing redirects here

Reply:
Oops, good point, thanks.